units/en/unit2/q-learning.mdx

```diff
@@ -18,74 +18,74 @@ The **Q comes from "the Quality" (the value) of that action at that state.**
 
 <img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit3/link-value-policy.jpg" alt="Link value policy"/>
 
 
-But, in the beginning, **our Q-Table is useless since it gives arbitrary values for each state-action pair** (most of the time, we initialize the Q-Table to 0). As the agent **explores the environment and we update the Q-Table, it will give us better and better approximations** to the optimal policy.
+But, in the beginning, **our Q-table is useless since it gives arbitrary values for each state-action pair** (most of the time, we initialize the Q-table to 0). As the agent **explores the environment and we update the Q-table, it will give us better and better approximations** to the optimal policy.
 
 <figure class="image table text-center m-0 w-full">
 <img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit3/Q-learning-1.jpg" alt="Q-learning"/>
-  <figcaption>We see here that with the training, our Q-Table is better since, thanks to it, we can know the value of each state-action pair.</figcaption>
+  <figcaption>We see here that with the training, our Q-table is better since, thanks to it, we can know the value of each state-action pair.</figcaption>
 </figure>
 
-Now that we understand what Q-Learning, Q-Function, and Q-Table are, **let's dive deeper into the Q-Learning algorithm**.
+Now that we understand what Q-Learning, Q-function, and Q-table are, **let's dive deeper into the Q-Learning algorithm**.
 
 ## The Q-Learning algorithm [[q-learning-algo]]
 
 This is the Q-Learning pseudocode; let's study each part and **see how it works with a simple example before implementing it.** Don't be intimidated by it, it's simpler than it looks! We'll go over each step.
 
 <img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit3/Q-learning-2.jpg" alt="Q-learning"/>
 
-### Step 1: We initialize the Q-Table [[step1]]
+### Step 1: We initialize the Q-table [[step1]]
 
 <img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit3/Q-learning-3.jpg" alt="Q-learning"/>
 
 
-We need to initialize the Q-Table for each state-action pair. **Most of the time, we initialize with values of 0.**
+We need to initialize the Q-table for each state-action pair. **Most of the time, we initialize with values of 0.**
 
-### Step 2: Choose action using epsilon greedy strategy [[step2]]
+### Step 2: Choose action using epsilon-greedy strategy [[step2]]
 
 <img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit3/Q-learning-4.jpg" alt="Q-learning"/>
 
 
 Epsilon greedy strategy is a policy that handles the exploration/exploitation trade-off.
 
-The idea is that we define epsilon ɛ = 1.0:
+The idea is that we define epsilon ɛ ≤ 1.0:
```

Comment:
Nice catch!

Reply:
I'm starting to think that I may have been mistaken on the epsilon value. I guess it's supposed to mean that the initial value is 1.0.

It's always ≤ 1.0, but it starts at 1.0

Maybe it's worth rephrasing to "we define the initial epsilon to ɛ = 1.0"

I can do the change, but I want to make sure I'm getting it right now.

Reply:
Yes, you're right. You initialize epsilon to 1 and then you have decay which will decrease the epsilon. Your rephrasing proposal sounds great!